### ISSUE TITLE
Reading System Guidance should be in a dedicated and separated document from EPUB Best Practices for Fixed Layout Accessibility

### ISSUE THREAD

**Issue by gautierchomel** [w3c/epub-specs#2660]
## Issue on the [“EPUB Fixed Layout Accessibility”](https://www.w3.org/TR/epub-fxl-a11y/) WG Note

Section 5. Fixed layout accessibility guidance for reading systems

## Describe the problem 

This section addresses a different audience (RS developers). 

## Describe the fix or new feature you propose

Create a new Group note document named "Fixed layout accessibility guidance for reading systems" and suppress section 5 from the document "EPUB Fixed Layout Accessibility"
